feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SessionItem.kt

```diff
@@ -73,6 +69,37 @@ class SessionItem @AssistedInject constructor(
         viewBinding.speakers.bindSpeaker()
     }
 
+    override fun bind(
+        viewBinding: ItemSessionBinding,
+        position: Int,
+        payloads: MutableList<Any>
+    ) {
+        if (payloads.isEmpty()) {
+            bind(viewBinding, position)
+        } else {
+            payloads.forEach { payload ->
```

Comment:
*payloads* would consist of `notifyItemChanged`'s payload and `notifyItemRangeChanged` payload. That's why the type of payloads is List. 

So `payloads.distinct` should be called in this design. Because binding would be called multiple times if multiple `ItemPayload.FavoritePayload` are included.